core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizer.java

```diff
@@ -2802,6 +2933,14 @@ private void triggerListeningToNamespace(final MongoNamespace namespace) {
       if (!nsConfig.isConfigured()) {
         return;
       }
```

Comment:
this is an odd change, but it had to made because this method is called from lots of places

Reply:
[cleanup] I think these should be handled in `this.configureSyncFrequency`. Looks like there is an opportunity here to merge these two methods.